internal/testcontext/context.go

```diff
@@ -20,7 +20,8 @@ import (
 	"storj.io/storj/internal/memory"
 )
 
-const defaultTimeout = 3 * time.Minute
+// DefaultTimeout is the default timeout used by new context
+const DefaultTimeout = 3 * time.Minute
```

Comment:
Why make this public?

Reply:
The reason is commented in the PR opening comment and in the commit message.

Reply:
Ah, sorry missed the description... I did the inital comments when you pushed the changes